sympy/physics/quantum/tests/test_grover.py

```diff
@@ -89,4 +89,4 @@ def test_grover():
     nqubits = 4
     basis_states = superposition_basis(nqubits)
     expected = (-13*basis_states)/64 + 264*IntQubit(2, nqubits)/256
-    assert apply_grover(return_one_on_two, 4) == qapply(expected)
+    assert apply_grover(return_one_on_two, 4) == qapply(expected, mul=True) # mul=True is default, to make terms ==
```

Comment:
I don't really see why `qapply` is being used in this test function at all. Is it not just possible to write `expected` in the "expected" form?

Reply:
This was the original test code. The original authors found it cumbersome to write 'expected' explicitly, and used qapply() - so it became a test case not only for apply_grover but also for qapply. I'm fine with that.